
bool server_running
bool is_recording
uint32 recording_time
uint16 port
string url
